build.gradle.kts

```diff
@@ -1,9 +1,23 @@
+buildscript {
+    repositories {
+        google()
+        mavenCentral()
+    }
+
+    dependencies {
+        classpath(Dependencies.Gradle.BUILD_KONFIG)
+        classpath(Dependencies.Kotlin.KOTLIN_GRADLE_PLUGIN)
+        classpath(Dependencies.Kotlin.KOTLIN_SERIALIZATION)
+    }
+}
```

Comment:
We don't define build plugins like this in Gradle 8 😄 Use the below `plugins` block instead 🙏

Reply:
For more information, I have a WIP PR in android-templates for the migration here https://github.com/nimblehq/android-templates/pull/508